Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss

```
@@ -1053,7 +1053,16 @@ Generic layout
 }
 
 #fancybox-inner {
-  @include buttonShadowBorder;
+  display: grid;
+  justify-content: center;
+}
+
+div#selection-content-inner.search-layout
+  div.mainCols
+  div#col2
+  div#selection-summary {
```

are the `div`s needed here? could they be omitted?

```suggestion
#selection-content-inner.search-layout
  .mainCols
  #col2
  #selection-summary {
```